### PR TITLE
Add Brownian noise examples, and fix CI

### PR DESCRIPTION
**Pull request by asinghvi17**
cc @mschauer @haampie

### CODE REVIEW

File: examples/examples3d.jl [hunk 1]
```diff
@@ -624,9 +624,62 @@
...
+        data = 0.1randn(d,d) + reshape(
+                cholesky(gridlaplacian(d,d) + 0.003I) \ randn(d*d), 
+                d, d
+        )
+        
+        surface(data; shading=false, show_axis=false, colormap = :deep)
+    end
+    
+    @cell "Coloured fractional Brownian noise field" ["3d"] begin
+        # This example was contributed by Harmen Stoppels (@harmen_stoppels)
```
Comment (haampie) [inline examples/examples3d.jl]: @haampie would be the correct handle :)

Comment (asinghvi17) [reply examples/examples3d.jl]:
```suggestion
        # This example was contributed by Harmen Stoppels (@haampie)

**Comment by asinghvi17** [comment]
@mschauer, would you mind giving some technical details about what exactly this is doing?

File: examples/diffeq.jl [hunk 2]
```diff
@@ -83,7 +83,8 @@ end
                         prob;
                         p = [σ, ρ, β],
                         tspan = (ts[1], ts[end])
-                    )
+                    ),
+                    Tsit5(),
```
Comment (mschauer) [inline examples/diffeq.jl]: Is this change intentional?

Comment (asinghvi17) [reply examples/diffeq.jl]: Yeah, had to make a couple fixes when I regenerated the reference image set, so I figured I'd chuck those in here as well.

File: examples/examples3d.jl [hunk 3]
```diff
@@ -625,8 +625,61 @@
         streamplot(f, -1.5..1.5, -1.5..1.5, -1.5..1.5, colormap = :magma, gridsize = (10, 10), arrow_size = 0.06)
     end
 
+    @cell "Fractional Brownian surface" ["3d"] begin
+        # This example was provided by Moritz Schauer (@mschauer).
+        using SparseArrays, LinearAlgebra
+
+        # Here, we define some form of noise function:
```
Comment (mschauer) [inline examples/examples3d.jl]:
```suggestion
        #=
               Define the precision matrix (inverse covariance matrix) for the Gaussian noise matrix It approximately coincides with the Laplacian of the 2d grid or the graph representing the
neighborhood relation of pixels in the picture, https://en.wikipedia.org/wiki/Laplacian_matrix 
        =#
```

File: examples/examples3d.jl [hunk 4]
```diff
@@ -625,8 +625,61 @@
         streamplot(f, -1.5..1.5, -1.5..1.5, -1.5..1.5, colormap = :magma, gridsize = (10, 10), arrow_size = 0.06)
     end
 
+    @cell "Fractional Brownian surface" ["3d"] begin
```
Comment (mschauer) [inline examples/examples3d.jl]:
```suggestion
    @cell "Brownian surface" ["3d"] begin
```

Comment (asinghvi17) [reply examples/examples3d.jl]: Dang...this will have to go in the next refimage update.  Will keep it in mind!

File: examples/examples3d.jl [hunk 5]
```diff
@@ -625,8 +625,61 @@
...
+                        end
+                    end
+                end
+            end
+            return S
+        end
+
+        # d is used to denote the size of the data
+        d = 150
+
```
Comment (mschauer) [inline examples/examples3d.jl]:
```suggestion

         # Sample centered Gaussian noise with the right correlation by the method
         # based on the Cholesky decomposition of the precision matrix
```

File: examples/examples3d.jl [hunk 6]
```diff
@@ -625,8 +625,61 @@
...
+        surface(data; shading=false, show_axis=false, colormap = :deep)
+    end
+
+    @cell "Coloured fractional Brownian noise field" ["3d"] begin
+        # This example was contributed by Harmen Stoppels (@haampie)
+        
+        using FFTW
+
+        function cloud(n = 256, p = 0.75f0)
```
Comment (mschauer) [inline examples/examples3d.jl]: # Obtain approximately fractal Brownian appropriately damping the high frequencies of Fourier transformed 
        # spatial white noise and (inverse) Fourier transform the result back into the spatial domain

File: examples/examples3d.jl [hunk 7]
```diff
@@ -629,7 +629,10 @@
         # This example was provided by Moritz Schauer (@mschauer).
         using SparseArrays, LinearAlgebra
 
-        # Here, we define some form of noise function:
+        #=
+               Define the precision matrix (inverse covariance matrix) for the Gaussian noise matrix It approximately coincides with the Laplacian of the 2d grid or the graph representing the
+neighborhood relation of pixels in the picture, https://en.wikipedia.org/wiki/Laplacian_matrix
```
Comment (asinghvi17) [inline examples/examples3d.jl]:
```suggestion
        neighborhood relation of pixels in the picture, https://en.wikipedia.org/wiki/Laplacian_matrix 
```

**Comment by asinghvi17** [comment]
Travis is dead, long live Travis
